deployment/portable/clean.sh

```diff
@@ -1,7 +1,27 @@
 #!/usr/bin/env bash
 
-source ../common.build.sh
+keep_artifacts="${1}"
 
-VERSION=`get_version ../..`
+WORKDIR="$( pwd )"
 
-clean_jellyfin ../.. Release `pwd`/dist/jellyfin_${VERSION}
+package_temporary_dir="${WORKDIR}/pkg-dist-tmp"
```

Comment:
I don't like that scripts are getting copied instead of re-used (like it was before via `clean_jellyfin`), why did you do this?

Reply:
I don't like these "common functions" in scripts. It makes each one unreadable and having to read back to somewhere else. They're also no longer independent. I like having everything each script needs inside of the script itself readable inline, despite duplication.